### PR TITLE
Support async completion

### PR DESCRIPTION
**Pull request by 333fred**
vscode side of https://github.com/OmniSharp/omnisharp-roslyn/pull/1986.

### CODE REVIEW

**Comment by 333fred** [comment]
@JoeRobich @filipw for review.

File: src/omnisharp/protocol.ts [hunk 1]
```diff
@@ -509,6 +510,7 @@ export interface QuickInfoResponse {
 export interface CompletionRequest extends Request {
     CompletionTrigger: CompletionTriggerKind;
     TriggerCharacter?: string;
+    UseAsyncCompletion: boolean;
```
Comment (333fred) [inline src/omnisharp/protocol.ts:513]:
```suggestion
```

**Review by JoeRobich — APPROVED** [review]

**Comment by 333fred** [comment]
@filipw could you take a look? It'd be good to get this in.

**Review by filipw — APPROVED** [review]
looks good to me! ✨

**Pull request merged**